src/http/server.ts

```diff
@@ -86,8 +91,21 @@ export class Server {
    * and the resource the HTTP request is trying to get is found, then
    * Drash.Http.Response will use its sendStatic() method to send the
    * static asset back to the client.
+   *
+   * TODO(crookse) Change this to a Map.
+   */
+  protected static_paths: string[] | { [key: string]: string } = [];
```

Comment:
Todo intentionally left in?

Reply:
yes. i don't plan to change it to a map in this PR as it doesn't relate. it's a note for a future issue